test/jdk/java/awt/event/ComponentEvent/TextFieldTextEventTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2007, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
If this is a newly created test, then the copyright should contain only the present year.

Reply:
I have added the original date based on the above comment that’s we get during the PR review below
https://github.com/openjdk/jdk/pull/7512#issuecomment-1075171066